apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-binary-checkbox.component.html

```diff
@@ -1,17 +1,17 @@
 <h3>Binary Checkbox in Template Driven Form</h3>
 <fdp-form-group [multiLayout]="true" [noLabelLayout]="false">
```

Comment:
This multi layout is obsolete, it was used to have single column (left zone) vs 5 zone/